### PR TITLE
Proposed appendix about "GeoJSON Text Sequences"

### PR DESCRIPTION
**Pull request by dret**

### CODE REVIEW

File: back.mkd [hunk 1]
```diff
@@ -179,6 +179,14 @@ the geometry objects described above:
           }]
       }
 
+
+# GeoJSON Text Sequences
+
+GeoJSON as defined in this specification consists of exactly one GeoJSON object. However, there may be cases in which application want to represent sets or sequences of GeoJSON objects. Representing sets or sequences of GeoJSON objects is outside of the scope of this specification.
```
Comment (tschaub) [inline back.mkd]: Either "an application wants to" or "applications want to".

Comment (dret) [inline back.mkd]: good catch. thanks.

File: back.mkd [hunk 2]
```diff
@@ -179,6 +179,14 @@ the geometry objects described above:
           }]
       }
 
+
+# GeoJSON Text Sequences
+
+GeoJSON as defined in this specification consists of exactly one GeoJSON object. However, there may be cases in which applications want to represent sets or sequences of GeoJSON objects. Representing sets or sequences of GeoJSON objects is outside of the scope of this specification.
+
+If such a representation is needed, a new media type is required that has the ability to represent sets or sequences. When defining such a media type, it may be useful to base it on "JSON Text Sequences" [RFC7464], leaving the foundations of how to represent multiple JSON objects to that specifications, and only defining how it applies to GeoJSON objects. However, no such definition is included here, and at the time of writing it is unclear if and when there will be a specification for "GeoJSON Text Sequences".
```
Comment (adoyle) [inline back.mkd]: "leaving the foundations of how to represent multiple JSON objects to that specifications"  -- specifications should be singular.

Comment (dret) [inline back.mkd]: thanks!

File: back.mkd [hunk 3]
```diff
@@ -179,6 +179,14 @@ the geometry objects described above:
           }]
       }
 
+
+# GeoJSON Text Sequences
+
+GeoJSON as defined in this specification consists of exactly one GeoJSON object. However, there may be cases in which applications want to represent sets or sequences of GeoJSON objects. Representing sets or sequences of GeoJSON objects is outside of the scope of this specification.
```
Comment (martinthomson) [inline back.mkd]: These need to be line wrapped to match the rest of the text.

**Comment by martinthomson** [comment]
I think that this also needs to modify template.xml to include the reference.

**Comment by dret** [comment]
call to merge this PR for closing #64: http://mailarchive.ietf.org/arch/msg/geojson/2BZDNzN1S5eRVJkmwAlikLFLUfo

**Comment by sgillies** [comment]
@dret the only change I see to make is to replace "sets or sequences" with "sequences" or "incrementally parseable sequences" since I think GeoJSON feature collections do already represent sets of feature objects.

**Comment by dret** [comment]
On 2016-01-30 19:17, Sean Gillies wrote:
> @dret <https://github.com/dret> the only change I see to make is to
> replace "sets or sequences" with "sequences" or "incrementally parseable
> sequences" since I think GeoJSON feature collections do already
> represent sets of feature objects.

my intention was to say that whether the "sequence" of JSON objects (as 
ordered in the JSON text sequence) should be relevant or not is up for 
the new media type to decide. it could be done either way. but maybe 
that's too much implicit meaning and we can make it simpler by just 
saying "sequences". if that seems to be the better route, then i am fine 
to go that way.

**Pull request merged**